.github/workflows/ci-unix-static-conformance.yml

```diff
@@ -11,61 +15,62 @@ concurrency:
 
     - uses: jwlawson/actions-setup-cmake@d06b37b47cfd043ec794ffa3e40e0b6b5858a7ec # v1.14.2
-      if: steps.cache-ext.outputs.cache-hit != 'true'
+      if: steps.cache-ext-with-libavif.outputs.cache-hit != 'true'
     - uses: ilammy/setup-nasm@13cbeb366c45c4379d3478cdcbadd8295feb5028 # v1.5.1
-      if: steps.cache-ext.outputs.cache-hit != 'true'
+      if: steps.cache-ext-with-libavif.outputs.cache-hit != 'true'
     - uses: seanmiddleditch/gha-setup-ninja@8b297075da4cd2a5f1fd21fe011b499edf06e9d2 # v4
-      if: steps.cache-ext.outputs.cache-hit != 'true'
+      if: steps.cache-ext-with-libavif.outputs.cache-hit != 'true'
     - run: pip install meson
-      if: steps.cache-ext.outputs.cache-hit != 'true'
+      if: steps.cache-ext-with-libavif.outputs.cache-hit != 'true'
 
     - name: Build dav1d
-      if: steps.cache-ext.outputs.cache-hit != 'true'
+      if: steps.cache-ext-with-libavif.outputs.cache-hit != 'true'
       working-directory: ./sys/dav1d-sys
       run: bash -e dav1d.cmd
     - name: Build libyuv
-      if: steps.cache-ext.outputs.cache-hit != 'true'
+      if: steps.cache-ext-with-libavif.outputs.cache-hit != 'true'
       working-directory: ./sys/libyuv-sys
       run: bash -e libyuv.cmd
 
+    - name: Clone av1-avif (specification repository for testFiles)
+      if: steps.cache-ext-with-libavif.outputs.cache-hit != 'true'
+      working-directory: ./third_party
+      run: |
+        git clone https://github.com/AOMediaCodec/av1-avif.git
+        cd av1-avif
+        git checkout 77bd20d59d7cade4ab98f276ff4828433ebd089b
+
+    - name: Build libavif (avifdec for conformance comparison)
+      if: steps.cache-ext-with-libavif.outputs.cache-hit != 'true'
+      working-directory: ./third_party
+      run: |
+        git clone https://github.com/AOMediaCodec/libavif.git
+        cd libavif
+        git checkout da70b0177613ae94b18a8467ca70b03d2276f099
+        cp -R ../../sys/dav1d-sys/dav1d ext/dav1d
+        cmake -S . -B build -DAVIF_CODEC_DAV1D=LOCAL -DAVIF_LIBYUV=OFF -DAVIF_BUILD_APPS=ON
+        cmake --build build --parallel -t avifdec
+
     - uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af # v1.0.7
       with:
         toolchain: stable
 
     - name: Build and run the Rust tests
-      run: cargo test -- --skip test_conformance
-
-    - name: Build GoogleTest
-      if: steps.cache-ext.outputs.cache-hit != 'true'
-      working-directory: ./third_party
-      run: bash -e googletest.cmd
-
-    - name: Build and run the C++ tests
-      # TODO: This step fails on macos. So run it on linux only for now.
-      if: runner.os == 'Linux'
-      run: |
-        cargo build --features capi --release
-        mkdir c_build
-        cd c_build
-        cmake ../c_api_tests/
-        make
-        make test
+      run: cargo test -- test_conformance --skip test_conformance::_166_expects
```

Comment:
this is expected. the file is actually invalid and i have a local fix for this which makes the test pass.

i will file a bug with av1-avif repo for this.

Reply:
Is there a tracking bug for that? This PR can be merged with a TODO.

Reply:
this is the bug: https://github.com/AOMediaCodec/av1-avif/issues/217